### ISSUE TITLE
Title is required to set notify popup dialog label

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-hotgrid#146]
### Subject of the issue
As per Core issue [Notify role="dialog" requires a label](https://github.com/adaptlearning/adapt-contrib-core/issues/483). The popup uses the `title` to set the `dialog` [label](https://github.com/adaptlearning/adapt-contrib-core/blob/6657b362f59cccbb71e49b0e306227fd7c25ddce/js/views/notifyPopupView.js#L17).

### Expected behaviour
`Title` should be set to required as without this, the popup `dialog` won't be properly identified by assistive technologies.
